Explorer/Assets/DCL/Rendering/Highlight/HighlightRendererFeature.cs

```diff
@@ -0,0 +1,253 @@
+﻿using DCL.Diagnostics;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace DCL.Rendering.Highlight
+{
+    public struct HighlightSettings
+    {
+        public Color Color;
+        public float Width;
+        public Vector3 Offset;
+    }
+
+    [Serializable]
+    internal class HighlightRendererFeature_Settings
+    {
+        // Parameters
+        [SerializeField] internal float OutlineThickness = 1.0f;
+        [SerializeField] internal float DepthSensitivity = 0.05f;
+        [SerializeField] internal float NormalsSensitivity = 1.0f;
+        [SerializeField] internal float ColorSensitivity = 0.5f;
+        [SerializeField] internal Color OutlineColor = new (1.0f, 1.0f, 1.0f, 0.5f);
+    }
+
+    public partial class HighlightRendererFeature : ScriptableRendererFeature
+    {
+        private const string k_ShaderName_HighlightInput = "DCL/HighlightInput_Override";
+        private const string k_ShaderName_HighlightInputBlur = "DCL/HighlightInput_Blur";
+        private const string k_ShaderName_HighlightOutput = "DCL/HighlightOutput";
+        private readonly ReportData m_ReportData = new ("DCL_RenderFeature_Outline", ReportHint.SessionStatic);
+
+        [SerializeField] private HighlightRendererFeature_Settings m_Settings;
+        public static Dictionary<Renderer, HighlightSettings> m_HighLightRenderers;
```

Comment:
Is there some way to avoid using a public static dictionary? Can it be injected somehow into the render feature and the `InteractionHighlightSystem`?